starfish/test/image/test_imagestack_coordinates.py

```diff
@@ -15,12 +15,12 @@
 WIDTH = 10
 
 
-def round_to_x(r: int) -> Tuple[float, float]:
-    return (r + 1) * 1000, (r + 1) * 100
+def round_to_x() -> Tuple[float, float]:
```

Comment:
I still have no idea why these are called what they are. 😆 

Could we just hard-code (1000, 100) and (10, 0.1)?

Finally, potentially not related to this, but you're our physical space expert, so: why are the coordinates (high, low)? I would have expected the opposite.

Reply:
yea we can, that's just how they were formatted before my changes and I was trying to change as little as possible